content/cloud/sqlite-storage.md

```diff
@@ -0,0 +1,105 @@
+title = "SQLite Storage"
+template = "cloud_main"
+date = "2023-07-26T16:00:00Z"
+enable_shortcodes = false
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/sqlite-storage.md"
+
+---
+
+- [Prerequisites](#prerequisites)
+- [Accessing Private Beta](#accessing-private-beta)
+- [Limitations For SQLite In Fermyon Cloud](#limitations-for-sqlite-in-fermyon-cloud)
+- [How To](#how-to)
+- [Create a Table In Your SQLite Database](#create-a-table-in-your-sqlite-database)
+- [Manage The SQLite Database Lifecycle](#managed-sqlite-database-lifecycle)
+- [Next Steps](#next-steps)
+
+With SQLite support in Fermyon Cloud, you can persist relational data generated by your Spin application in between application invocations without having to worry about database management. Fermyon Cloud will provision and manage the database on your behalf. To learn more about the Spin SQLite SDK, please visit the [API guide](../spin/sqlite-api-guide.md).
+
+## Accessing Private Beta
+
+SQLite Storage in Cloud is currently in private beta. To request access to the private beta, please fill out this short [sign-up form](https://fibsu0jcu2g.typeform.com/to/Brv12FI0#hubspot_utk=xxxxx&hubspot_page_name=xxxxx&hubspot_page_url=xxxxx).
+ 
+> Please note that the private beta is limited in space and all requests cannot be guaranteed. 
+
+## Service Limitations and Quotas For SQLite in Fermyon Cloud
+
+*Service Limitations*
+* There is a 1:1 relationship between Spin application and SQLite Store
+* You must delete your Spin application before you can delete your SQLite Store
+* The `--sqlite` flag not supported on `spin cloud deploy`
+* Fermyon Cloud doesn't support "custom" SQLite Stores
+
+*Quotas* 
+* You can have a maximum of 1 "default" SQLite Store 
+* Your SQLite Store can hold up 1 GB of data
+
+## How To:
+
+### Prepare a SQLite Database
+
+Since `--sqlite` is not supported on `spin cloud deploy`, we recommend using the `execute` command in your application's source code to create a table. For example, in this routing function we use the Rust SDK to check if the table exists. If it doesn't, then we create one:
```

Comment:
@rylev's been working on an import command using with `spin cloud sqlite` but it will only work at application runtime (not during deployment). I think there's a way we adjust the wording though to make it a bit less abrasive to the user.